pkg/event/target/amqp.go

```diff
@@ -148,9 +149,18 @@ func (target *AMQPTarget) Close() error {
 
 // NewAMQPTarget - creates new AMQP target.
 func NewAMQPTarget(id string, args AMQPArgs) (*AMQPTarget, error) {
-	conn, err := amqp.Dial(args.URL.String())
-	if err != nil {
-		return nil, err
+	var conn *amqp.Connection
+	var err error
+	// Retry 10 times with time interval of 2 seconds.
+	for i := 1; i <= 10; i++ {
+		conn, err = amqp.Dial(args.URL.String())
+		if err == nil {
+			break
+		}
+		if err != nil && i == 10 {
+			return nil, err
+		}
+		time.Sleep(2 * time.Second)
```

Comment:
IMHO 20 secs wait is a little long for user to wait for server start up (in case RabbitMQ isn't available/reachable). We can bring it to 10 secs (2s wait with 5 retries)

Reply:
Done.